app/views/shared/views/ems_common/angular/_form.html.haml

```diff
@@ -97,14 +97,30 @@
         %span.help-block{"ng-show" => "angularForm.azure_tenant_id.$error.isUuid"}
           = _("Invalid input format, please enter a GUID")
 
-    .form-group{"ng-class" => "{'has-error': angularForm.subscription.$invalid}", "ng-if" => "emsCommonModel.emstype != '' && emsCommonModel.emstype == 'azure'"}
+    .form-group{"ng-class" => "{'has-error': angularForm.azure_tenant_id.$invalid}", "ng-if" => "emsCommonModel.emstype != '' && emsCommonModel.emstype == 'azure_stack'"}
+      %label.col-md-2.control-label{"for" => "ems_tenant_id"}
+        = _('Tenant ID')
+      .col-md-4
+        %input.form-control{"type"                    => "text",
+                            "id"                      => "ems_tenant_id",
+                            "name"                    => "azure_tenant_id",
+                            "ng-model"                => "emsCommonModel.azure_tenant_id",
+                            "required"                => "",
+                            "checkchange"             => "",
+                            "prefix"                  => "default",
+                            "reset-validation-status" => "default_auth_status"}
+        %span.help-block{"ng-show" => "angularForm.azure_tenant_id.$error.required"}
+          = _("Required")
+
+    .form-group{"ng-class" => "{'has-error': angularForm.subscription.$invalid}", "ng-if" => "emsCommonModel.emstype != '' && (emsCommonModel.emstype == 'azure' || emsCommonModel.emstype == 'azure_stack')"}
```

Comment:
same here, the ng-if could just `['azure', 'azure_stack'].includes(emsCommonModel.emstype)`